modules/installation-about-mirror-registry.adoc

```diff
@@ -13,3 +13,8 @@ The mirror registry can be any container registry that supports link:https://doc
 The mirror registry must be reachable by every machine in the clusters that you provision. If the registry is unreachable installation, updating, or normal operations such as workload relocation might fail. For that reason, you must run mirror registries in a highly available way, and the mirror registries must at least match the production availability of your {product-title} clusters.
 
 When you populate a mirror registry with {product-title} images, you can follow two scenarios. If you have a host that can access both the internet and your mirror registry, but not your cluster nodes, you can directly mirror the content from that machine. This process is referred to as _connected mirroring_. If you have no such host, you must mirror the images to a file system and then bring that host or removable media into your restricted environment. This process is referred to as _disconnected mirroring_.
+
+[NOTE]
```

Comment:
My personal pref is to avoid the admonitions unless it seems critical.  In this case, the presence of the note kind of makes the reader anticipate that there's some trouble or something problematic.  Maybe:
* Try it as a plain para.
* Can you follow this spot in the assembly with a `.Additional resources` block title and add an xref to the procedure?

As with everything else, up to you.